src/Init/Prelude.lean

```diff
@@ -3342,17 +3370,22 @@ end
 section
 variable {ρ : Type u} {m : Type u → Type v}
 
-/-- `(← read) : ρ` gets the read-only state of a `ReaderT ρ`. -/
+/-- Retrieves the reader monad's local value. -/
 @[always_inline, inline]
 protected def read [Monad m] : ReaderT ρ m ρ :=
   pure
 
-/-- The `pure` operation of the `ReaderT` monad. -/
+/--
+Returns the provided value, ignoring the reader monad's local value.
+-/
 @[always_inline, inline]
 protected def pure [Monad m] {α} (a : α) : ReaderT ρ m α :=
   fun _ => pure a
 
-/-- The `bind` operation of the `ReaderT` monad. -/
+/--
+Sequences two reader monad computations. Both are provided with the local value, and the second is
+passed the value of the first.
```

Comment:
With the Nat docstrings, there was guidance to use, eg, `/` over `Nat.div`. Do you want to say something similar here, to use `>>=` over `bind` (maybe with a comment about universe polymorphism)?

Reply:
Thanks! I'll add the `>>=` note.

I think the universe comment would be a bit too much here, but if we encounter users who are confused by it then we can reconsider.